README.md

```diff
@@ -316,18 +316,31 @@ Navigate to Kubernetes directory and update the yamls with latest docker image o
 
 For DEV and PROD branches, automatically deployed latest yaml with latest agent image (which automatically built by the azure devops pipeline) onto CIDEV and CIPROD AKS clusters in build subscription.  So, you can use CIDEV and CIPROD AKS cluster to validate E2E. Similarly, you can set up build and release pipelines for your feature branch.
 
+# Testing MSI Auth Mode Using Yaml
+
+  1. Enable Monitoring addon with Managed Idenity Auth Mode either using Portal or CLI or Template
+  2. Deploy [ARM template](./scripts/onboarding/aks/onboarding-using-msi-auth/) with enabled = false to create DCR, DCR-A and link the workspace to Portal
+   > Note - Make sure to update the parameter values in existingClusterParam.json file and have enabled = false in template file
+    `az deployment group create --resource-group <ResourceGroupName> --template-file ./existingClusterOnboarding.json --parameters @./existingClusterParam.json`
```

Comment:
Are you referring to these files here: 
https://raw.githubusercontent.com/microsoft/Docker-Provider/gangams/fbit-customizations/scripts/custom-configuration/fbit/existingClusterOnboarding.json
https://raw.githubusercontent.com/microsoft/Docker-Provider/gangams/fbit-customizations/scripts/custom-configuration/fbit/existingClusterParam.json

Might be worth including links to make this more clear

Reply:
> Are you referring to these files here: https://raw.githubusercontent.com/microsoft/Docker-Provider/gangams/fbit-customizations/scripts/custom-configuration/fbit/existingClusterOnboarding.json https://raw.githubusercontent.com/microsoft/Docker-Provider/gangams/fbit-customizations/scripts/custom-configuration/fbit/existingClusterParam.json
> 
> Might be worth including links to make this more clear

> No. These files in the branch itself. if you see the step#2, there is link for the template directory.